svelte-kit-scss/src/lib/components/IssueSearch/IssueSearchListItem/IssueSearchListItem.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import type { Issue } from '$lib/interfaces';
+  import IssueStateIcon from './IssueStateIcon.svelte';
+  import IssueStateDescription from './IssueStateDescription.svelte';
+  import CommentsCount from './CommentsCount.svelte';
+  import IssueSearchAssignees from './IssueSearchAssignees.svelte';
+  import IssueLabel from './IssueLabel.svelte';
+
+  export let item: Issue;
+  $: ({ title, commentsCount, state, assignees, labels } = item);
+</script>
+
+<div class="issue-search-list-item">
+  <div class="state">
+    <IssueStateIcon {state} />
+  </div>
+  <div class="main">
+    <div class="title">
+      <a href="#" class="link-anchor">
```

Comment:
I assume this is one of those missing features? Do we need an issue view to link to?